src/common/AsyncCallbackWrap.h

```diff
@@ -38,29 +39,27 @@ enum asyncCallBackType {
 
 struct AsyncCallbackWrap {
  public:
-  AsyncCallbackWrap(AsyncCallback* pAsyncCallback, MQClientAPIImpl* pclientAPI);
+  AsyncCallbackWrap(AsyncCallback *pAsyncCallback, MQClientAPIImpl *pClientAPI);
   virtual ~AsyncCallbackWrap();
-  virtual void operationComplete(ResponseFuture* pResponseFuture,
-                                 bool bProducePullRequest) = 0;
+  virtual void onComplete(ResponseFuture *pResponseFuture) = 0;
   virtual void onException() = 0;
   virtual asyncCallBackType getCallbackType() = 0;
 
  protected:
-  AsyncCallback* m_pAsyncCallBack;
-  MQClientAPIImpl* m_pClientAPI;
+  AsyncCallback *m_pAsyncCallBack;
+  MQClientAPIImpl *m_pClientAPI;
 };
 
 //<!************************************************************************
 class SendCallbackWrap : public AsyncCallbackWrap {
  public:
-  SendCallbackWrap(const string& brokerName, const MQMessage& msg,
-                   AsyncCallback* pAsyncCallback, MQClientAPIImpl* pclientAPI);
+  SendCallbackWrap(const string &brokerName, const MQMessage &msg,

```

Comment:
remove `using namespace std;` and use `std::string` here.

Reply:
Nice catch, best to avoid namespace pollution.

Reply:
> using namespace std;
in another head file.